include/SDL3/SDL_stdinc.h

```diff
@@ -1216,6 +1216,52 @@ extern SDL_DECLSPEC int SDLCALL SDL_vswprintf(SDL_OUT_Z_CAP(maxlen) wchar_t *tex
 extern SDL_DECLSPEC int SDLCALL SDL_asprintf(char **strp, SDL_PRINTF_FORMAT_STRING const char *fmt, ...) SDL_PRINTF_VARARG_FUNC(2);
 extern SDL_DECLSPEC int SDLCALL SDL_vasprintf(char **strp, SDL_PRINTF_FORMAT_STRING const char *fmt, va_list ap) SDL_PRINTF_VARARG_FUNCV(2);
 
+/**
+ * Seed the pseudo-random number generator
+ *
+ * \param seed the value to use as a random number seed, or 0 to use SDL_GetPerformanceCounter()
```

Comment:
Personally, I wouldn't initialize with `SDL_GetPerformanceCounter`, and expect users to do `SDL_srand(SDL_GetPerformanceCounter());` themselves.

Reply:
Why require that extra step? It's not like people are going to be using a specific value except in special circumstances.

Reply:
My idea about the SDL stdlib functions is that they behave (more-or-less) 1:1 as the libc functions.
When not using srand, glibc's rand function is 100% reproducible due to the seed initialized to 0.
This won't be the case with SDL.

Reply:
But [reading the spec](https://pubs.opengroup.org/onlinepubs/9699919799/functions/rand.html), the initial state is left unspecified.

Reply:
Actually, it does specify it:
`If rand() is called before any calls to srand() are made, the same sequence shall be generated as when srand() is first called with a seed value of 1.`

However, we already use a 64-bit seed value, so I'm fine with the SDL API diverging in this case for the sake of convenience. If somebody does want a fixed seed value, they can just call srand() with that seed before they start.

Reply:
Haha, you're pulling a Uno Reverse here. Nice one :)